src/Voron/Data/Fixed/FixedSizeTree.cs

```diff
@@ -1553,5 +1553,75 @@ public override string ToString()
         {
             return Name.ToString();
         }
+
+        public long GetNumberOfEntriesAfter(long value, out long totalCount)
+        {
+            totalCount = NumberOfEntries;
+            if (totalCount == 0)
+                return 0;
+
+            if (Type.HasValue == false)
+                return 0;
+
+            long count = 0;
+            if (Type.Value == RootObjectType.EmbeddedFixedSizeTree)
+            {
+                using (var it = Iterate())
+                {
+                    if (it.Seek(value) == false)
+                        return 0;
+
+                    do
+                    {
+                        if (it.CurrentKey == value)
+                            continue;
+
+                        count++;
+                    } while (it.MoveNext());
```

Comment:
Why are you doing this iteration? You already know the position in the value, and the # of entries, you can just compute it.